src/icons/help-center.svg

```diff
@@ -0,0 +1 @@
xat1ML5S1Nqc94gr6bUOMu27IB0F1imVbKhoVdSgpA/xXoipcapLbUQeKqWUgMVmjfIQ4HKUbbtIfBASk3NG+R2NE7DJlLawtW35Y7qiwxXYt4gL6GoheGKU84tIDAvpTbmDcK28ZkIhQjYEFiMRHfbJIw1TS+D8LWNy2KtsOoVNQHuO/26xqOufK/K9TIIeyF0FOlFS/P6EXgLC8/qt0ITl/UyCNsxE0rqm4CNUgQlMBu58+cHkooig/B3Ms5MqqVqTEgCLyLz40MWUFfeRQZhfX4MLeYHhQgMIHA0lts+rjwg63oX9zMIhwwcCPHcUiECRQR+hwW/LlrY9Pn9DMK28abPIZBuHpKGIk/gNsz4NpTs/jHIIARyMcR3ReminTQUIwQW4cPe0GsjM1L8b2IQtvs8aH9Io31JQzEfCD4H8Zeckg5TgxDCH6CdoDs5oWgtgZPR8mkQf3o5+ShjEMLgSM2doR9CrQCEdiqGCdyDfzxqHAu9PTwr/b9lDUIihPMLiM9v8720SyBFugQeRNO+AU2E/gwpShJYFevvAXGYAW8wskdDajaDx7ENz4F4xLD5EkWy2oOXAjb7YWa4TP6RW5tW8YhybUdMz4ptzw+RxHOox9xI6hJjNfhrvvxFWF5wPwTxVGoxxJ2r9eHDIDFD5EafArFLUiECpQk09fBp0tAXsJLMYUJK6xQSSNUgHDxHc9xV2HItEAEDAikahObYHdL9GoMdQKv0J5CaQV5Cc6dCf+/fbC0VATMCKRmENy5pjoVmTddaIjCYQCoGYTclzZHUK2cGbz6tEZpACgahOTg2iO/0UoiAVwJNNwgf6uJdfD6XoBAB7wSabBAOvac5bvVORRmKQIdAUw3Ch3S+AP1VW1IEQhJookFoDv4ex80hwShvESCBphnkddR5L0jDrrn1FMEJNGmw4og5bvJEZVvkM6mT16X4z1G/ChHIEGiKQd5Arb8ILcjU3n7i40g6D1q3kwUv9GWQDgz9W0GgCadYS1FdmuPGFdV2+pQ3h1NmSpw2gdgNQnPsA93gaTPIHJ5AtiWbmA3CX9zdF7re08aQOTyBbFM2IQzi491ZNMd+0HWeNobM4Qlk27IJYZD7HSHyXcDTIT7n7iNkDh8UW5pHCINc5cCS5pgBXeOQR3dSmaObhj6XJhDCIFeiFo+Wrsnw+7b2RzoXg3UXK3N009BnKwIhDMLXAB0O8XUypsE0B0B/Mk0wYD2ZYwAgLTYjEMIgLHk+dCDEu9+DgjcBaY4rBq1ouFzmMASl1eonsCWqwFOmojfc8f7Gdh6rSXPwXVhF5RXN39FjHZRVHATuttgPuH+c0V390ENNHkZhHFw4AeKzG+Mhxj8h3t+4lxOeYmPk0z18pEy2m2BlPctehljc666E6nGb2oSP2xQ25QZPMw4lFB0hBs0/PXjtVECVBHZw2BeOqrKiVZa1Kgpjx8AgM/RazufaN6iysiorKIELkXuv7Wwyjzeok4070TITCL3WuRppQ59yJgs+ooZ9zWEf4H7BM5Fk42y0rNfObzqPHQobJksn7YatjObx9Mj2LIL7CH/2IROpfWNyBz8i08JyE3yUlx0L50O84H8GWgYpyhFYhNU5KsI2PoyE6xkk5sX42hB7Ib8ObQO5xB9dEjch7RqoJK8nTI8YWi8Mq99jG6zmsMPMqmkbfjZf51A3CvPlVDX9OgqaU1VhKqeQwHQsuQhyMUlh5oEW8Kg35kUgqRmE7Nhlq9Mikqg3mmaSk4CLZxSZSNEgvIY4N9NKTdRFoCkm4U3iS+uCVEe566PQZyFdY8TBoOw1SZXXIDzbmFi0k6Z4BGFbaY7vFDVa8ysnEPOR5HjQaO3vyfwSjddRJB4GpkeSqo4gHEHO+yetDd7n4bmlTBIPAxOTVGEQ9ljxtkDrg12NcyGZJB4Gg0wS2iC8Cbxm653RBYCH0VMhmSQeBv1MEtIgv8J+0KT7M127cfiP+6CI/0AyShwMikwSwiDsuDkg/C7W/BI+gCbwZiLHCsko9TPoZRKfBuHbOU+DuN0VJQhsinVPgZZAMkq9DPIm8WEQPrU6E9JzPoDgEuzpmgqdDN0CvQjJMNUz6DaJi0GOwfbj6F6O9HUOL5k41yKuDMiEh+N1IPZ0pHozFU1zjuOQg8/z+suR30HQmdBhkE1on7ahpjRBCKyKXH13ofNIMgeyPYoHaagyFQFbAiFM8g4qI4PYbhGli45ACJPIINFtZlXIhUAsJnFpg9KKQFACMZgkaAOVuQi4EqjbJK71V3oRCE6AJrkMsr2OcEkXvHEqQAR8EKjLJD7qrjxEoBICdZikkoapEBHwRaBqk/iqt/IRgcoIVGmSyhqlgkTAJwE+pFTFhbvPOisvEaiUQBUmqbRBKkwEfBMIbRLf9VV+IlA5gZAmqbwxKlAEQhAIZZIQdVWeIlALAZqEz3+43DnPp62lISpUBEIR8GkSPkeiEIHkCPgyyeLkyKhBItAh4MMkfFRXIQLJEnA1ya7JklHDRKBDwNYkfHmEQgRaQYAmKTMshb8vqDcntmLXUCNHCPCF43zvFl8dmu/K7Z6+BMtlDkBQtJPApmj2idBC6DWIXbmPQedDkyCFCIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIhA5AT+D4DtlW6R4VO7AAAAAElFTkSuQmCC"/><mask id="d" height="15" maskUnits="userSpaceOnUse" width="15" x="0" y="0"><path d="m.5.25h14.25v14.25h-14.25z" fill="url(#b)"/></mask></defs><path d="m.5.25h14.25v14.25h-14.25z" fill="url(#a)" opacity=".720959"/><g mask="url(#d)"><path d="m.5.25h14.25v14.25h-14.25z" fill="#9293ab"/></g></svg>
```

Comment:
have you passed to svgo? https://harness.atlassian.net/wiki/spaces/PLATFORM/pages/1963262066/SVG+Guidelines+Best+Practices